ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml

```diff
@@ -26,19 +26,26 @@ tests:
   container:
     from: src
 - as: test-integration
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 2h0m0s
+    version: "4.7"
```

Comment:
do we care? will this pool also exist in the future?

Reply:
It might not, not sure why it's there but I am keeping the original configuration it was before just so nothing pops up